src/chrome/komodo/content/colorpicker/colorpicker.js

```diff
@@ -367,7 +367,7 @@ Refresh.Web.ColorPicker.prototype = {
 				break;
 				
 			default:
-				alert('invalid mode');
+				console.error('invalid mode');

```

Comment:
These are not valid fixes. Agreed alert is crude but it provides actual user feedback. `console` calls are not allowed in production code.